packages/arb-token-bridge-ui/src/components/common/Notifications.tsx

```diff
@@ -57,8 +57,8 @@ function NitroDevnetNotification() {
 }
 
 export function Notifications() {
-  const { l1 } = useNetworksAndSigners()
-  const { isGoerli } = isNetwork(l1.network.id)
+  const [{ sourceChain }] = useNetworks()
+  const { isGoerli } = isNetwork(sourceChain.id)
```

Comment:
on prod we use l1 so when user switches between goerli and arb goerli on the tx panel, the notification stays

but here it disappears when the source chain is not goerli

let's change the variable from `isGoerli` to `isTestnet`, that'd make more sense and also include sepolia & arb sepolia